.gitignore

```diff
@@ -18,7 +18,6 @@
 
 **/*project.lock.json
 **/project.lock.json
-**/*.nuspec
```

Comment:
This doesn't look correct. The `nuspec` files are supposed to be ignored.

Reply:
Reverted.